sleap/gui/dialogs/formbuilder.py

```diff
@@ -224,25 +225,32 @@ def get_results(self) -> Optional[Dict[Text, Any]]:
         return self._results
 
 
-# We are writing this method to override the spinbox in add_item so the increment/decrement is x10/x0.1
 class ExponentialSpinBox(QtWidgets.QDoubleSpinBox):
+    """We are writing this method to override the spinbox in add_item so the increment/decrement is x10/x0.1"""
```

Comment:
We aren't concerned with what WE are doing, but rather what the class is:
```suggestion
    """A subclass of QDoubleSpinBox that increments and decrements by a factor of 10."""
```

> All class docstrings should start with a one-line summary that describes what the class instance represents.

Ref: [google-style docstrings](https://google.github.io/styleguide/pyguide.html#384-classes)